geomesa-quickstart-python/tools/dbModify/insert.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+#coding:utf-8
+"""
+insert.py
+
+Description:
+   These functions prepare data and insert them into Accumulo tables (GeoMesa SimpleFeatures).
+
+Created by: Jordan Muss
+
+Creation Date: 4-12-2017
+Version:       1.0
+
+Dependencies: 
+         Public:     jnius, json
+         Private:   ECQL, setupJnius
+
+Interfaces:
+            queryFeaturesToDict         something
+                                                     more
+
+Updates:
+
+To Do:
+"""
+from __future__ import print_function
+
+def insert(JNI, ecql, simpleFeatureTypeName, dataStore, filter_string, field_dict, print_num=10):
+    ''' Insert data into an accumulo simpleFeatureType (table): '''
+    jLong = JNI.autoclass('java.lang.Long')
+    WKTUtils = JNI.autoclass("org.locationtech.geomesa.utils.interop.WKTUtils")
+    dfc = JNI.autoclass("org.geotools.feature.DefaultFeatureCollection")
+    SimpleFeatureStore = JNI.autoclass("org.geotools.data.simple.SimpleFeatureStore")
+    
+    # ToDo: This is test stuff, delete this block when finished
```

Comment:
Still working on wrapping the test code. This was just a place holder. Either I need to finish it, or remove it and make a branch of my branch